opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbDao.java

```diff
@@ -4730,24 +4734,42 @@ public void configureWifiRrm(OvsdbClient ovsdbClient, OpensyncAPConfig opensyncA
                 freqBand = "5G";
             }
 
+            
             ElementRadioConfiguration elementRadioConfig = apElementConfig.getRadioMap().get(radioType);
-            if (elementRadioConfig == null) {
+            RfElementConfiguration rfElementConfig = rfConfig.getRfConfig(radioType);
```

Comment:
Please check rfConfig for null, before calling any of it's accessors, and if it is null, then continue, similar to what you are doing with the rfElementConfig etc.